### PR TITLE
newrelic-infra-agent 1.42.0

### PR DESCRIPTION
**Pull request by rogercoll**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/newrelic-infra-agent.rb [hunk 1]
```diff
@@ -42,8 +42,8 @@ def install
   end
 
   def post_install
-    (etc/"newrelic-infra").mkpath
-    (var/"log/newrelic-infra").mkpath
+    # (etc/"newrelic-infra").mkpath
+    # (var/"log/newrelic-infra").mkpath
```
Comment (p-linnane) [inline Formula/newrelic-infra-agent.rb:46]: Why are we commenting this out?

Comment (rogercoll) [reply Formula/newrelic-infra-agent.rb:46]: Good catch! This PR was done automatically by bump-formula tool.

I will push a commit to fix it. Thanks

**Comment by chenrui333** [comment]
can you just squash the commits and forced update this PR?

File: Formula/newrelic-infra-agent.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/newrelic-infra-agent.rb]: It would be a bit shocking that `bump-formula-pr` command would comment out any code.

Comment (chenrui333) [reply Formula/newrelic-infra-agent.rb]: This is the cmd code, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb

**Review by p-linnane — APPROVED** [review]

**Pull request merged**